src/Integration/Suppression/SonarQubeIssuesProvider.cs

```diff
@@ -215,6 +215,7 @@ private async Task SynchronizeSuppressedIssuesAsync()
                     .ToDictionary(x => x.Key, x => x.RelativePathToRoot);
                 this.hasModules = moduleKeyToRelativePathToRoot.Keys.Count > 1;
                 this.allSuppressedIssues = await this.sonarQubeService.GetSuppressedIssuesAsync(sonarQubeProjectKey,
+                    null, // TODO - pass the current branch
```

Comment:
Fix up the product code so it doesn't fail (I've manually tested with the slvs-bound sample that suppressions and taint issues are still fetched).